speedy-recovery-client/src/components/core/App.js

```diff
@@ -228,6 +263,24 @@ class App extends Component {
   };
 
   resetError = () => this.setState({ error: null });
+
+  deleteRepeatingElement = resource => {
```

Comment:
This is quite a long/inefficient way of doing this. What about doing something like this:
``removeArrayDuplicates = array => array !== undefined ? array.reduce((prev, curr) => prev.find(a => a["text"] === curr["text"]) ? prev : prev.push(curr) && prev, []) : array;``

Reply:
Wow! great!